Makefile

```
@@ -104,4 +105,29 @@ merge_coverage_results:
 .ONESHELL:
 run_token_server_integration_tests:
 	pip3 install -r tools/tokenserver/requirements.txt
-	pytest tools/tokenserver --junit-xml=${INTEGRATION_JUNIT_XML}
\ No newline at end of file
+	pytest tools/tokenserver --junit-xml=${INTEGRATION_JUNIT_XML}
+
+.ONESHELL:
+run_mysql_e2e_tests:
+	docker compose \
+		-f docker-compose.mysql.yaml \
+		-f docker-compose.e2e.mysql.yaml \
+	 	up \
+	 	--exit-code-from mysql-e2e-tests \
+	 	--abort-on-container-exit;
+	exit_code=$$?
+	docker cp mysql-e2e-tests:/mysql_integration_results.xml ${LOCAL_INTEGRATION_JUNIT_XML}
+	exit $$exit_code
+
+.ONESHELL:
+run_spanner_e2e_tests:
```

I'd like to make it clear that these invoke docker, let's prefix them with `docker_` similar to the target `docker_start_mysql` and the like (and maybe they should live in that section of the file too).

While you're there could you also change them to invoke `docker compose` (w/ a space) instead of the old `docker-compose`?
```suggestion
docker_run_mysql_e2e_tests:
	docker compose \
		-f docker-compose.mysql.yaml \
		-f docker-compose.e2e.mysql.yaml \
	 	up \
	 	--exit-code-from mysql-e2e-tests \
	 	--abort-on-container-exit;
	exit_code=$$?
	docker cp mysql-e2e-tests:/mysql_integration_results.xml ${LOCAL_INTEGRATION_JUNIT_XML}
	exit $$exit_code

.ONESHELL:
docker_run_spanner_e2e_tests:
```